src/include/sof/audio/module_adapter/module/generic.h

```diff
@@ -207,6 +207,11 @@ struct processing_module {
 	 */
 	bool skip_src_buffer_invalidate;
 
+	/*
+	 * flag to insure that module is loadable
+	 */
+	bool is_loadable;
```

Comment:
Is this used in the previous commit ? if so, needs squashed.

Reply:
done